src/TrueSheet.tsx

```diff
@@ -250,7 +251,7 @@ export class TrueSheet extends PureComponent<TrueSheetProps, TrueSheetState> {
         scrollableHandle={this.state.scrollableHandle}
         sizes={sizes}
         blurTint={blurTint}
-        background={backgroundColor}
+        background={(backgroundColor ? processColor(backgroundColor) : undefined) as any}
```

Comment:
Maybe change the `background` type in `TrueSheetNativeViewProps` instead of casting to `any`

Reply:
If you change this property in TrueSheetNativeViewProps so that its type is number instead of ColorType, the user won't be able to use it as a regular color property. I don't see any problem with casting to any because it's part of the internal implementation and won't be exposed externally.

Reply:
the `background` prop is actually just a prop for the native component. The user facing prop is `backgroundColor`.